internal/probe/ebpf/detector.bpf.c

```diff
@@ -543,18 +531,124 @@ int tracepoint__syscalls__sys_enter_openat(struct syscall_trace_enter* ctx) {
         }
 
         if (compare_open_filenames(&opened_filename, configured_filename)) {
-            process_event_t event = {
-                .type = PROCESS_FILE_OPEN,
-                .pid = *userspace_pid,
-            };
-            bpf_perf_event_output(ctx, &events, BPF_F_CURRENT_CPU, &event, sizeof(event));
+            bpf_map_update_elem(&file_opens_by_pid, &pid, &dummy_val, BPF_ANY);
             return 0;
         }
     }
 
     return 0;
 }
 
+SEC("tracepoint/syscalls/sys_enter_open")
+int tracepoint__syscalls__sys_enter_open(struct syscall_trace_enter* ctx) {
+    u64 pid_tgid = bpf_get_current_pid_tgid();
+    u32 pid =  (u32)(pid_tgid & 0xFFFFFFFF);
+    u32 tgid = (u32)(pid_tgid >> 32);
```

Comment:
Maybe add an explicit comment on what tgid and oid stand for in kernel vs uspace